coins/cosmos.json

```diff
@@ -2,7 +2,7 @@
   {
     "platform":"cosmos",
     "denom":"uatom",
-    "symbol":"atom",
+    "symbol":"ATOM",
```

Comment:
- symbol should lowercase
- struct of coin should be same 
- "denom": "ibc/" is invalid